docker-compose.yml

```diff
@@ -0,0 +1,33 @@
+name: "otel-profiling"
+
+services:
+  agent:
+    build:
+      context: .
+      args:
+        - arch=${ARCH:?error}
+    privileged: true
+    pid: "host"
+    network_mode: "host"
+    volumes:
+      - ~dd/otel-profiling-agent:/agent
```

Comment:
I was surprised to see `~dd` work here, why not use `./` instead ?

Reply:
oh wow good catch, sorry I missed this when looking at the PR, and must have copy-pasted this part from the commands I was running manually (which work locally due to my workspace's setup)